.github/CONTRIBUTING.md

```diff
@@ -80,10 +80,10 @@ For example, to contribute to `langchain` run `cd libs/langchain` before getting
 To install requirements:
 
 ```bash
-poetry install -E all
+poetry install
```

Comment:
Thanks! we definitely don't want `all`, we do want contributors to have the test group installed to run linters and local unit tests

```suggestion
poetry install --with test
```